### ISSUE TITLE
Get a copy of the WDL submitted

### ISSUE THREAD

**Issue by sitapriyamoorthi** [getwilds/proof#26]
In the old version of PROOF one was able to get a copy of a WDL that was run. 
It would be nice if this was still available!

**Comment by sckott** [comment]
@sitapriyamoorthi can you please describe what you mean? where exactly

**Comment by sitapriyamoorthi** [comment]
Where can I get a copy of my WDL workflow that o submitted from the app!

**Comment by sckott** [comment]
yes i understand the ask, but where was it in the old app, i don't see it
